kratos/tests/auxiliar_files_for_python_unittest/vtk_output_process_ref_files/ascii3D/Main.vtu

```diff
@@ -0,0 +1,44 @@
+<VTKFile type="UnstructuredGrid" version="0.1" byte_order="LittleEndian">
+   <UnstructuredGrid>
+      <Piece NumberOfPoints="15" NumberOfCells="13">
+         <Points>
+            <DataArray type="Float64" Name="Position" NumberOfComponents="3" format="ascii">
+              0.000000000e+00  1.000000000e+00  1.000000000e+00  0.000000000e+00  1.000000000e+00  0.000000000e+00  0.000000000e+00  0.000000000e+00  1.000000000e+00  1.000000000e+00  1.000000000e+00  1.000000000e+00  0.000000000e+00  0.000000000e+00  0.000000000e+00  1.000000000e+00  1.000000000e+00  0.000000000e+00  1.000000000e+00  0.000000000e+00  1.000000000e+00  1.000000000e+00  0.000000000e+00  0.000000000e+00  2.000000000e+00  1.000000000e+00  1.000000000e+00  2.000000000e+00  1.000000000e+00  0.000000000e+00  2.000000000e+00  0.000000000e+00  1.000000000e+00  2.000000000e+00  0.000000000e+00  0.000000000e+00  0.000000000e+00  0.000000000e+00  2.000000000e+00  1.000000000e+00  0.000000000e+00  2.000000000e+00  1.000000000e+00  1.000000000e+00  2.000000000e+00
+            </DataArray>
+         </Points>
+         <Cells>
+            <DataArray type="Int32" Name="connectivity" NumberOfComponents="1" format="ascii">
+              11  9  7  8  3  5  8  6  10  6  8  7  4  2  7  5  3  5  6  2  1  2  4  5  9  8  5  7  6  7  2  5  6  7  5  8  3  0  5  2  8  11  10  7  2  1  0  5  2  6  3  12  13  14
+            </DataArray>
+            <DataArray type="Int32" Name="offsets" NumberOfComponents="1" format="ascii">
+              4  8  12  16  20  24  28  32  36  40  44  48  54
+            </DataArray>
+            <DataArray type="UInt8" Name="types" NumberOfComponents="1" format="ascii">
+              10  10  10  10  10  10  10  10  10  10  10  10  13
+            </DataArray>
+         </Cells>
+         <PointData>
+            <DataArray type="Float64" Name="DISPLACEMENT" NumberOfComponents="3" format="ascii">
+              0.000000000e+00  3.000000000e+00  4.000000000e+00  0.000000000e+00  3.000000000e+00  0.000000000e+00  0.000000000e+00  0.000000000e+00  4.000000000e+00  2.000000000e+00  3.000000000e+00  4.000000000e+00  0.000000000e+00  0.000000000e+00  0.000000000e+00  2.000000000e+00  3.000000000e+00  0.000000000e+00  2.000000000e+00  0.000000000e+00  4.000000000e+00  2.000000000e+00  0.000000000e+00  0.000000000e+00  4.000000000e+00  3.000000000e+00  4.000000000e+00  4.000000000e+00  3.000000000e+00  0.000000000e+00  4.000000000e+00  0.000000000e+00  4.000000000e+00  4.000000000e+00  0.000000000e+00  0.000000000e+00  0.000000000e+00  0.000000000e+00  8.000000000e+00  2.000000000e+00  0.000000000e+00  8.000000000e+00  2.000000000e+00  3.000000000e+00  8.000000000e+00
```

Comment:
Question, can we remove not necessary significative values, or the test will fail?

Reply:
Can and done :)